.github/workflows/test-suite.yml

```diff
@@ -57,37 +57,23 @@ jobs:
       matrix:
         terraform_version: ${{ fromJSON(needs.versions.outputs.terraform_matrix) }}
         provider_version: ${{ fromJSON(needs.versions.outputs.provider_matrix) }}
-    name: mig-tests-${{ matrix.terraform_version || 'latest' }}-${{ matrix.provider_version || 'latest' }}
-    secrets: inherit
-    uses: ./.github/workflows/migration-tests.yml
-    with:
-      terraform_version: ${{ matrix.terraform_version }}
-      provider_version: ${{ matrix.provider_version }}
-
-  acc-tests:
-    needs: [clean-before, versions]
-    if: ${{ !cancelled() }}
-    strategy:
-      max-parallel: 1
-      fail-fast: false
-      matrix:
-        terraform_version: ${{ fromJSON(needs.versions.outputs.terraform_matrix) }}
-    name: acc-tests-${{ matrix.terraform_version || 'latest' }}
+    name: tests-${{ matrix.terraform_version || 'latest' }}-${{ matrix.provider_version || 'latest' }}
     secrets: inherit
     uses: ./.github/workflows/acceptance-tests.yml
     with:
       terraform_version: ${{ matrix.terraform_version }}
+      provider_version: ${{ matrix.provider_version }}
       atlas_cloud_env: ${{ inputs.atlas_cloud_env }}
```

Comment:
[q] If the test suite is executed with a list of provider versions (for migration tests), acceptance tests would only be run if the value "" is also included right?

Reply:
correct